ejcIMC.py

```diff
@@ -25,6 +25,25 @@ def imcCalculate(statureValue, weigthValue):
     IMC = (weigthValue/(statureValue**2))
     return IMC
 
+def yourImc(imc):
+    valueImc = imc
```

Comment:
why u declarate valueImc if u dont use in the function?

Reply:
ok, i will delete valueImc